src/main/kotlin/com/supergrecko/questionbot/services/ConfigService.kt

```diff
@@ -21,6 +21,7 @@ data class QGuild(
         val guild: Guild
 ) {
     fun getQuestion(id: Int) = config.questions.first { it.id == id }
+    fun getAnser(questionId: Int) = {}
```

Comment:
Could you fix this typo?

Reply:
Fixed, thanks - I didn't actually need that function in the end so just removed it instead.